docs/Developers/Testing_Bears.rst

```diff
@@ -24,8 +24,9 @@ Let us understand how to write tests for ``TooManyLinesBear`` in ``some_dir``.
 
     class TooManyLinesBear(LocalBear):
 
-        def run(file,
+        def run(self,
```

Comment:
why break lines here when you have so much space

Reply:
It was already like that in docs. I was just following the same pattern. You want me to change it, but that change will reflect in docs also.